aries_cloudagent/protocols/present_proof/v2_0/manager.py

```diff
@@ -387,6 +393,8 @@ async def verify_pres(self, pres_ex_record: V20PresExRecord):
                 ).verify_pres(
                     pres_ex_record,
                 )
+                if pres_ex_record.verified == "false":
```

Comment:
Maybe this should probably specifically check for "true" instead of only failing on the expected failure value?

Reply:
> Maybe this should probably specifically check for "true" instead of only failing on the expected failure value?

The verified status for the record is set according to the verification status of the lastly verified format.

The idea was to stop the iteration (for-loop for each format) in case there is a failing verification. So if there appears a record with failed verification, the result of the entire presentation verification will stay "false" (pres_ex_record.verified == "false") without checking next formats. The resond behind this logic is that initially the problem was that if after a failed verification there was a format with a successful one, the verified status was overwritten and set to "true".

Is this understandable? Any ideas one how to improve this are welcomed.

Reply:
Okay, seems fine if it's just short-circuiting and not potentially accepting a failed verification.